packages/create-toolpad-app/src/index.ts

```diff
@@ -87,116 +65,136 @@ async function isFolderEmpty(root: string, name: string): Promise<boolean> {
 // Detect the package manager
 const packageManager = getPackageManager();
 
-// Install the dependencies
-const installDeps = async (projectName: string, cwd: string) => {
-  const { execaCommand } = await import('execa');
+const validatePath = async (relativePath: string): Promise<boolean | string> => {
+  const { default: chalk } = await import('chalk');
+
+  const absolutePath = path.join(process.cwd(), relativePath);
+
+  try {
+    await fs.mkdir(absolutePath);
```

Comment:
```suggestion
    await fs.mkdir(absolutePath, { recursive: true });
```

Reply:
From https://nodejs.org/api/fs.html#fspromisesmkdirpath-options 
> Calling fsPromises.mkdir() when path is a directory that exists results in a rejection only when recursive is false

I want to be able to detect when a directory exists on calling `mkdir` so that I am able to catch that error and then call `isDirectoryEmpty` on the existing directory, but setting `recursive` to `true` causes me to miss that case

Reply:
Won't you need `recursive` if you pass a nested directory to the CLI? e.g. 

```
yarn create toolpad-app some/nested/dir
```

I'd just validate the path before calling `fs.mkdir` instead of after